src/components/Form.tsx

```diff
@@ -91,13 +91,18 @@ export class UnwrappedForm extends Component<IFormWrappedProps> {
   }
 
   public render() {
-    const { showControls, title } = this.props,
+    const { showControls, title, layout, colon } = this.props,
       formModel = this.formManager.formModel,
       filteredFieldSets = filterFieldSets(this.fieldSets, { model: formModel }),
       className = cx(CLASS_NAME, this.props.className);
 
     return (
-      <Antd.Form layout="vertical" onSubmit={this.formManager.onSave} className={className}>
+      <Antd.Form
+        layout={layout}
+        onSubmit={this.formManager.onSave}
+        className={className}
+        colon={colon === false ? false : true}
```

Comment:
With prop defaults, you can just do this, right?
```suggestion
        colon={colon}
```